bin/format-admonitions.rb

```diff
@@ -0,0 +1,8 @@
+BACKTICK_MATCH_REGEX = /(?<start>`{3,})(?<type>exercism\/[a-z]+)(?<admonition>[\s\S]*?)(?<end>\k<start>)/.freeze
```

Comment:
I love your use of named captures.

`[\s\S]` is exactly `.` -- i.e. any character.

Reply:
But I think maybe that was when I was using it in a lookahead.

Reply:
It doesn't work. I've just double-checked

Reply:
Yeah - so this is what I was running into when testing on https://regex101.com/

<img width="427" alt="Screenshot 2023-09-20 at 16 43 20" src="https://github.com/exercism/problem-specifications/assets/286476/e62f68b7-1df4-41bd-9f57-3894a5316560">

(Sample regexp if you want to play: 
````
(?<before>[\s\S]*)(?<start>```)(?<type>exercism\/[a-z]+\n(?<admonishion>.*?))(?<end>```)(?<after>[\s\S]*)````

Reply:
> Interestingly, this didn't work for me. `.` didn't include newlines.

You need the `m` option for the regex: https://ruby-doc.org/3.2.2/Regexp.html#class-Regexp-label-Options

Reply:
> Yeah - so this is what I was running into when testing on https://regex101.com/

Have to be a bit careful with regex testing sites. Are Ruby regexes PCRE?

Reply:
> You need the m option for the regex: https://ruby-doc.org/3.2.2/Regexp.html#class-Regexp-label-Options

Nice! @ErikSchierboom's finished for the day but can update it to this tomorrow :)

> Have to be a bit careful with regex testing sites. Are Ruby regexes PCRE?

🤷 Internet seems to say yes.

Reply:
Fixed in https://github.com/exercism/problem-specifications/pull/2327/commits/388ed5278588299f30d11b19d5516d0b9def3266